<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillEditorClass</class>
 <widget class="QMainWindow" name="SkillEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkillEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <item>
     <widget class="IIRenderWindow" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddItem"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>TimeBar</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,9">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="TimeBar" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAddItem">
   <property name="icon">
    <iconset>
     <normaloff>../../../../素材区/PNGico090418/NetByte Design Studio - 0022.png</normaloff>../../../../素材区/PNGico090418/NetByte Design Studio - 0022.png</iconset>
   </property>
   <property name="text">
    <string>addItem</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TimeBar</class>
   <extends>QWidget</extends>
   <header>timebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IIRenderWindow</class>
   <extends>QGraphicsView</extends>
   <header>renderwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/skilleditor.qrc"/>
 </resources>
 <connections/>
</ui>
